pkg/agent/multicast/mcast_discovery.go

```diff
@@ -137,10 +137,18 @@ func (s *IGMPSnooper) validate(event *mcastGroupEvent) (bool, error) {
 }
 
 func (s *IGMPSnooper) validatePacketAndNotify(event *mcastGroupEvent) {
-	allow, _ := s.validate(event)
-	if allow {
-		s.eventCh <- event
+	allow, err := s.validate(event)
+	if err != nil {
+		// Antrea Agent does not remove the Pod from the OpenFlow group bucket directly,
```

Comment:
```suggestion
		// Antrea Agent does not remove the Pod from the OpenFlow group buckets immediately when an error is returned,
		// but it will be removed if no valid IGMP report is received before Controller.mcastGroupTimeout.
```

Reply:
done